packages/rulesets/src/arazzo/index.ts

```diff
@@ -0,0 +1,258 @@
+      },
+    },
+    'arazzo-step-parameters-validation': {
+      description: 'Step parameters and workflow parameters must valid.',
+      message: `{{error}}`,
+      recommended: true,
+      severity: 0,
+      given: '$',
+      then: {
+        function: arazzoStepParametersValidation,
+      },
+    },
+    'arazzo-step-failure-actions-validation': {
+      description:
+        'Every failure action must have a unique "name", and the fields "workflowId" and "stepId" are mutually exclusive.',
+      message: `{{error}}`,
+      recommended: true,
+      severity: 0,
+      given: '$',
+      then: {
+        function: arazzoStepFailureActionsValidation,
+      },
+    },
+    'arazzo-step-success-actions-validation': {
+      description:
+        'Every success action must have a unique "name", and the fields "workflowId" and "stepId" are mutually exclusive.',
+      message: `{{error}}`,
+      recommended: true,
+      severity: 0,
+      given: '$',
+      then: {
+        function: arazzoStepSuccessActionsValidation,
+      },
+    },
+    'arazzo-workflow-depends-on-validation': {
+      description: 'Every workflow dependency must be valid.',
+      recommended: true,
+      severity: 0,
+      given: '$',
+      then: {
+        function: arazzoWorkflowDependsOnValidation,
+      },
+    },
+    'arazzo-step-success-criteria-validation': {
+      description: 'Every success criteria must have a valid context, conditions, and types.',
+      message: `{{error}}`,
+      recommended: true,
+      severity: 0,
+      given: '$.workflows[*]',
+      then: {
+        function: arazzoStepSuccessCriteriaValidation,
+      },
+    },
+    'arazzo-step-request-body-validation': {
+      description: 'Every step request body must have a valid `contentType` and use of runtime expressions.',
+      recommended: true,
+      severity: 0,
+      given: '$',
+      then: {
+        function: arazzoStepRequestBodyValidation,
+      },
+    },
+    'arazzo-step-validation': {
+      description:
+        'Every step must have a valid "stepId" and an valid "operationId" or "operationPath" or "workflowId".',
+      recommended: true,
+      severity: 0,
+      given: '$',
+      then: {
+        function: arazzoStepValidation,
+      },
+    },
+    'arazzo-no-script-tags-in-markdown': {
+      description: 'Markdown descriptions must not have "<script>" tags.',
+      recommended: true,
+      given: '$..[description,title]',
+      then: {
+        function: pattern,
+        functionOptions: {
+          notMatch: '<script',
+        },
+      },
+    },
+    'arazzo-info-description': {
+      description: 'Info "description" should be present and non-empty string.',
+      recommended: true,
+      severity: 'warn',
+      given: '$',
+      then: {
+        field: 'info.description',
+        function: truthy,
+      },
+    },
+    'arazzo-info-summary': {
+      description: 'Info "summary" should be present and non-empty string.',
```

Comment:
Description says should be present but it is a hint? Maybe change it to something like its recommended to have a summary?